docs/cse/automation-service/automation-service-use-cases.md

```diff
@@ -0,0 +1,173 @@
+   1. Click the Actions button.
+   1. Under **Insight Automation**, select the automation you created in the previous step (it will have the same name as the playbook). The playbook runs.
+   1. To see the results of the run, click the **Automations** tab at the top of the Insight.
+   1. View the **Status** field to find out if the playbook has a status of Success or another status such as **Completed with errors**. 
+   1. Click **View Playbook** to see details of the playbook run. Each node in the playbook will show either **Success** or **Failed**. 
+   1. Click a node to download results of that node’s run.
+
+## Custom use case: Configure a custom action to run a local script
+
+The following example shows how to create an integration with a custom action that runs a script you provide. The custom action uses [IP Quality Score](https://www.ipqualityscore.com/) to gather IP reputation information for enrichment. 
+
+1. [Install the Automation Service Bridge](/docs/cse/automation-service/automation-service-bridge/). Because this example uses a custom action, you must first install the Bridge before you proceed.
+1. Obtain an API key from IP Quality Score:
+    1. Create a free account on [IP Quality Score](https://www.ipqualityscore.com/create-account).
+    1. Log in.
+    1. Go to your [account settings](https://www.ipqualityscore.com/user/settings) and  copy the **API Key**. You will use this key later.
+1. Create a new IP Quality Score integration:
+    1. Click the **Configuration** button (gear icon) at the top of the UI.
+    1. Under **Integrations**, select **Automation**.
+    1. From the Automation screen, click **Manage Playbooks**.
+    1. Click **Integrations** in the navigation menu.
+    1. Click the **+** icon at the top of the screen to the left of **Integrations**. The **New Integration** dialog appears. 
+    1. Download this file: <a href="https://help.sumologic.com/files/IP-Quality-Score-Test.yaml" target="_blank">IP-Quality-Score-Test.yaml</a>   
+    1. Drag the file into the **Select File** box in the **New Integration** dialog.
+    1. Click **Upload**. An IP Quality Score integration is created.
+    1. Open the new **IP Quality Score** integration. 
+    1. Hover your mouse over the **IP Quality Score** name and click the **Upload** button that appears.
+    1. In the **Upload** dialog, select **Action** in the **kind** field. 
+    1. Download this file: <a href="https://help.sumologic.com/files/IP-Reputation.yaml" target="_blank">IP-Reputation.yaml</a> 
+    1. Drag the file into the **Select File** box in the **New Integration** dialog. 
+    1. Click **Upload**. The **IP Reputation** action appears in the IP Quality Score integration.
+1. Add the IP Quality Score integration resource:
+    1. Click the **+** button to the left of **Resources**. 
+    1. Fill out the **Add Resource** dialog:
+    * **Label**: Enter **IP Quality Score Resource**.
+    * **API URL**: Enter **https://www.ipqualityscore.com/**
```

Comment:
Is the forward slash at the end of the URL intentional? Asking because I know it can be for API.

```suggestion
    * **API URL**: Enter `https://www.ipqualityscore.com/`.
```